src/delimiter.js

```diff
@@ -318,6 +313,112 @@ const makeStackedDelim = function(delim, heightTotal, center, options, mode,
         Style.TEXT, options, classes);
 };
 
+const sqrtInnerSVG = {
+    main : `<svg viewBox=\'0 0 400000 1000\' preserveAspectRatio=\'xMinYMin
```

Comment:
Escaping isn't necessary in template strings.  This could be:
```
main: `<svg viewBox='0 0 400000 1000' preserveAspectRatio='xMinYMinslice'>
...
</svg>`
```
nit: we don't put a space before the colon in the rest of the code.